### ISSUE TITLE
BridgeTalk functions are undefined

### ISSUE THREAD

**Issue by SteveLewisZA** [Adobe-CEP/CEP-Resources#396]
I'm trying to work with BridgeTalk in an CEP panel and don't seem to be able to get it working, through CSInterface I'm executing the Extendscript code:

`var bt = new BridgeTalk;`

All BridgeTalk functions on bt appear to be undefined (bt.send() etc.) am I doing something wrong?

Steve

**Comment by sujaisivanandan** [comment]
@SteveLewisZA BridgeTalk is not supported anymore. Can you look at Vulcan.js?
https://github.com/Adobe-CEP/CEP-Resources/blob/master/CEP_11.x/Vulcan.js

**Comment by SteveLewisZA** [comment]
@sujaisivanandan Oh I see, I'm trying to send Extendscript code from an After Effects extension to be executed in Media Encoder, is this possible with Vulcan.js?

**Comment by sujaisivanandan** [comment]
Yes. can you check the last section of https://fenomas.com/2014/09/cep-5-events-en/

**Comment by SteveLewisZA** [comment]
@sujaisivanandan So to communicate with Media Encoder would I need to have an extension installed on Media Encoder? Is it possible to install an extension on Media Encoder? I can't seem to find what the HostList name for Media Encoder would be in manifest.xml

Any help much appreciated

**Comment by sujaisivanandan** [comment]
@SteveLewisZA the host ID's are available here: https://github.com/Adobe-CEP/CEP-Resources/blob/master/CEP_11.x/Documentation/CEP%2011.1%20HTML%20Extension%20Cookbook.md#applications-integrated-with-cep

However as you rightly pointed out is mainly for communication between two extensions who have a defined event handlers. This may not suite your use-case. Let me check what are the other options for you.
